code/src/components/StartScreen.js

```diff
@@ -0,0 +1,49 @@
+import React, { useState } from 'react'
+
+import { directions, fetchStart } from '../reducers/directions'
+
+import NeonCat from './NeonCat'
+
+import { useDispatch } from 'react-redux'
+
+const StartScreen = () => {
+  const [name, setName] = useState('')
+  const dispatch = useDispatch()
+
+  const onInputChange = (event) => {
+    setName(event.target.value)
+  }
+
+  const onBtnClick = () => {
+    dispatch(directions.actions.setPlayer(name))
+    dispatch(fetchStart())
+  }
+
+  return (
+    <div className="nes-container with-title is-centered">
+      <p className="title">NEON CAT GAME!</p>
+      <NeonCat />
+      <div className="nes-field">
+        <label htmlFor="name_field">Enter your name to start the game:</label>
+        <input 
+          type="text" 
+          id="name_field" 
+          className="nes-input" 
+          value={name} 
+          onChange={onInputChange} 
+        />
```

Comment:
Nice feature with adding a unique id to the username! ✨

One thing we noticed though is that it's possible to enter the game without typing in a username. One suggestion is to make it required (or another kind of validation), and an appropriate message to the user on what to do, preventing the user from proceeding on. 🙂